README.md

```diff
@@ -11,10 +11,12 @@ with the Mbed OS.
 
 ### Development environment
 
-Please refer to `vagrant/bootstrap.sh` and `vagrant/bootstrap-user.sh` for
-details on how to set up a development environment. These scripts can be run
-locally on Linux, or you may use Vagrant to create a VM suitable for
-development (see `vagrant/README.md` for instructions).
+We have provided a ready-to-use Vagrant virtual machine for building
```

Comment:
Squash with vagrant changes commit.

Reply:
I've squashed it and updated the commit message